app-android/build.gradle.kts

```diff
@@ -3,20 +3,16 @@ plugins {
     kotlin("android")
     id("org.jetbrains.compose")
     id("kotlin-parcelize")
-    id("social.androiddev.code-quality")
 }
 
-val targetSDKVersion: Int by rootProject.extra
-val minSDKVersion: Int by rootProject.extra
-val compileSDKVersion: Int by rootProject.extra
-
 android {
     namespace = "social.androiddev.dodo"
-    compileSdk = compileSDKVersion
+
+    compileSdk = libs.versions.android.compileSdk.get().toInt()
 
     defaultConfig {
-        minSdk = minSDKVersion
-        targetSdk = targetSDKVersion
+        minSdk = libs.versions.android.minSdk.get().toInt()
```

Comment:
Slightly uglier access but it keeps the version info in libs.versions.toml like other versions, what do you think?

Reply:
I won't lose sleep over it lol but how about updating the ext in project gradle to still have minSDK, etc but instead of hardcoded values, it references these versions from the catalog? At least the ugliness will be contained to a single place that way. Don't feel strongly either way.

Reply:
It's only used in 2 places now so kinda meh about it, honestly might be better to figure out a way to just define it once in the convention plugins.